platform/engineering/backend/review-instances.md

```diff
@@ -37,6 +37,58 @@ You will need your browser configured to access the vetsgov-internal domain via
 1. Specify the branch names for `api_branch` and `web_branch`. These branches will be deployed together with the review instance.
 1. When the process is completed, the URL for the review instance will be provided at the end of the output logs.
 
+### Access and Usage
+
+**NOTE:** This section is actively being worked on right now and these processes should get easier as time goes on. Some of these processes are extremely painful at the moment, but we are making the feature available in its rough form as these get worked out. The epic for this work can be found [here](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/4617), please reach out to #vfs-platform-support for feature requests or questions.
+
+These instances are running on an independent virtual machine that includes an installation of both vets-api and vets-website with an nginx proxy in front getting requests to the right place.
+
+Anyone who is cleared by their COR to work on VSP and has already has access to the internal tools should be able to access these instances via SSH. Once logged in, you can modify the code already checked out and manipulate the review instances as necessary.
+
+Note that each time a commit is pushed against the branch the instance was automatically provisioned from, an attempt will be made to update the instance to the latest version from that branch. This could result in lost changes if you develop both on the instance and locally.
+
+Both vets-website and vets-api are started and managed via Docker Compose using the `docker-compose.review.yml` at the top level of their repository. To check either of them:
+
+- `ssh dsva@<hostname>`
+- `cd <vets-api|vets-website>`
+- `docker-compose -f docker-compose.review.yml ps`
+
+To get the `hostname` use these steps to get the correct hostname:
+
+- click the "Deployed" link in the GitHub deployment panel to get to the Jenkins job
+- click on "Console output" to view the provisioning task
+- search for "Review instance available at" on the page, which will give you the hostname string for SSH access
```

Comment:
Might want to add "See vets-api docker containers" & "See vets-website docker containers" to the "Common tasks" section, and list them as:
`cd vets-api; docker-compose -f docker-compose.review.yml ps` & `cd vets-website; docker-compose -f docker-compose.review.yml ps` respectively.

That way, we can have a dedicated "SSH to the Review Instance" section that just which is probably what people will really want to know. As it is now, it's kind of buried.

Reply:
Also, just realized, wouldn't it be better to just `docker ps` to see images?

Reply:
re: `docker ps`, yeah that's one way of doing it.. you'll see everything running on the instance and we may end up with more than just vets-api and vets-website.

Reply:
Adding a dedicated SSH section is a good idea.